### PR TITLE
Add ro_rootfs metric in a tmpfs folder

### PR DESCRIPTION
**Pull request by hectorhuertas**
* Removing dead code in `common.tf`
* Only for workers: mount the textfile collector dir as tmpfs and setup a timer that checks the ro status of the rootfs and updates a metric accordingly

### CODE REVIEW

File: resources/hector.service [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Hector test
```
Comment (ffilippopoulos) [inline resources/hector.service]: You are not using this one as far as I can see

File: resources/prometheus-ro-rootfs.service [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[Unit]
```
Comment (ffilippopoulos) [inline resources/prometheus-ro-rootfs.service]: Does that work without `[Install]`? Do we run it anywhere already so I can see it in action?

Comment (hectorhuertas) [reply resources/prometheus-ro-rootfs.service]: This service is not "installed". It is started by the timer, that by default starts a unit with the same name but being .service. It is similar to the behaviour of the systemd_service_restarter module.

File: resources/prometheus-ro-rootfs.timer [hunk 3]
```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Run prometheus-ro-rootfs service periodically
+After=prometheus-tmpfs-dir.service
+Requires=prometheus-tmpfs-dir.service
+
+[Timer]
+OnBootSec=60
+OnUnitInactiveSec=10
```
Comment (george-angel) [inline resources/prometheus-ro-rootfs.timer]: We scrape every 30s, so I don't think we'll benefit from this kind of precision. Set it to `30s` as well?

**Pull request merged**